TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Views/NodeInfoView.js

```diff
@@ -9,7 +9,19 @@ function(
 	template
 ) {
 	Ember.Handlebars.registerBoundHelper('formatDate', function(value) {
-		return new Date(value).toISOString().slice(0, 16).replace('T', ' ');
+		function pad(n) {
+			return n < 10 ? '0' + n : n;
+		}
```

Comment:
this is too easy! this would be a great time to add a dependency on the infamous https://www.npmjs.com/package/left-pad 😄